### PR TITLE
Use FQCN when creating a form on sf3

### PR DESCRIPTION
**Pull request by acrobat**
I am targeting this branch, because of a sf3 incompatibility

## Changelog

```markdown
### Fixed
- Use FQCN when creating a form on sf3
```

## Subject

Use FQCN when creating a form on sf3 as creating a form by name is deprecated in symfony 2.x. An incompatibility discovered when upgrading to sf3 and I want to create a new snapshot

### CODE REVIEW

**Comment by acrobat** [comment]
Fixed @greg0ire!

File: Controller/SnapshotAdminController.php [hunk 1]
```diff
@@ -46,7 +47,11 @@ public function createAction(Request $request = null)
 
         $snapshot->setPage($page);
 
-        $form = $this->createForm('sonata_page_create_snapshot', $snapshot);
+        $formType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix') ?
+            CreateSnapshotType::class :
+            'sonata_page_create_snapshot';
+
+        $form = $this->createForm($formType, $snapshot);
```
Comment (jlamur) [inline Controller/SnapshotAdminController.php]: We stopped supporting SF < 2.8 so you can go for:

```
$form = $this->createForm(CreateSnapshotType::class, $snapshot);
```

Comment (acrobat) [reply Controller/SnapshotAdminController.php]: yes indeed, changes were made before #877 was merged. But fixed now!

**Comment by greg0ire** [comment]
Please rebase, and StyleCI shall allow the `::class` notation

**Comment by acrobat** [comment]
Done @greg0ire

**Comment by greg0ire** [comment]
StyleCI is happy now :)

**Pull request merged**